.gitignore

```diff
@@ -179,4 +179,7 @@ dist
 cipherstash-proxy.toml
 
 # turbo repo
-.turbo
\ No newline at end of file
+.turbo
+
+# ignore release folder
+release/
```

Comment:
Might be worth checking in the release DSL file and pointing individual language examples to it. We could also add a check in CI that the generated file is up to date.

Reply:
that's a good idea. I'll update to commit it.

I'll do the check in CI separately as want to get back to the poc search_doc testing.